docs/content/users/quickstart.md

```diff
@@ -194,6 +194,48 @@ ddev launch
 
     When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
 
+## Grav
+
+=== "Composer"
+
+    ```bash
+    mkdir grav
+    cd grav
+    ddev config --php-version=8.2 --omit-containers=db
+    ddev start
+    ddev composer create getgrav/grav
+    ddev exec bin/gpm install admin -y
+    ddev launch
+    ```
+
+=== "Git Clone"
+
+    ```bash
+    mkdir grav
+    cd grav
+    git clone -b master https://github.com/getgrav/grav.git .
+    ddev config --php-version=8.2 --omit-containers=db
+    ddev start
+    ddev composer install --no-dev -o
+    ddev exec bin/grav install
+    ddev exec bin/gpm install admin -y
```

Comment:
I think we need to install `require-dev` dependencies, and add `ddev launch`.

```suggestion
    ddev composer install
    ddev exec bin/grav install
    ddev exec bin/gpm install admin -y
    ddev launch
```

Reply:
good catch with the missing `ddev launch`.  about the composer install line. i've used the `--no-dev -0` because those are the options used in the official grav docs: https://learn.getgrav.org/17/basics/installation (see option 3). therefore i stuck to those in this case. i am fine changing it to the variant you've suggest (i would have used the same) but i thought the grav folks added those for a reason and therefore used that when i created the PR. that is the background. but just wanted to let you know that info before committing the suggestions. so just let me know if i should still go with the changes for the composer line and adding the launch line or if i should only add the launch line?

Reply:
Given that DDEV is a local development tool, the `require-dev` dependencies should always be installed.
I think there's a mistake in their docs (at least they *should* mention that these flags are usually used for production).

Reply:
yep makes total sense. but just wanted to make sure. committed your proposed suggestion. thank you for taking a look and testing!